src/chain/shared_state.rs

```diff
@@ -40,7 +40,9 @@ pub struct SharedState {
     /// This is not a `BTreeSet` as it is ordered according to the sequence of pushes into it.
     pub our_infos: NonEmptyList<EldersInfo>,
     /// Info about all members of our section - elders, adults and infants.
-    pub our_members: BTreeMap<PublicId, MemberInfo>,
+    pub our_members: BTreeMap<XorName, MemberInfo>,
+    /// members that we had before last split.
+    pub our_old_members: BTreeMap<XorName, MemberInfo>,
```

Comment:
Maybe better call this `sibling_members` ?

Reply:
I'll update to `pre_split_sibling_members`

Reply:
Well, technically they are `post_split` siblings  as they weren't siblings before the split. Naming is hard :)

Reply:
So I think just `sibling_members` should be enough. Don't think it's ambiguous as that term is used only when referring to splits.

Reply:
They are only the sibling member at the time of the split, after that the actual sibling members will change, and we keep just a snapshot.